airbyte-integrations/connectors/source-declarative-manifest/source_declarative_manifest/run.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+import argparse
+import json
+import pkgutil
+import sys
+from typing import List
+
+from airbyte_cdk.connector import BaseConnector
+from airbyte_cdk.entrypoint import AirbyteEntrypoint, launch
+from airbyte_cdk.models import AirbyteMessage, ConnectorSpecification, Type
+from airbyte_cdk.sources.declarative.manifest_declarative_source import ManifestDeclarativeSource
+
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
```

Comment:
nit: is this part of one of our templates? If so, we should update the template at some point. it seems out of date

Reply:
Removing this in favor of just the small 2023 license, but we have a surprising amount of these bit 2020 MIT licenses floating around in the codebase

Reply:
fixed in [9c26936](https://github.com/airbytehq/airbyte/pull/36501/commits/9c269366fa1a7e34e45e22bcfce9150fed614c09)